integration-cli/docker_cli_run_unix_test.go

```diff
@@ -205,12 +205,20 @@ func (s *DockerSuite) TestRunWithCpuPeriod(c *check.C) {
 func (s *DockerSuite) TestRunWithKernelMemory(c *check.C) {
 	testRequires(c, kernelMemorySupport)
 
-	dockerCmd(c, "run", "--kernel-memory", "50M", "--name", "test", "busybox", "true")
+	dockerCmd(c, "run", "--kernel-memory", "50M", "--name", "test1", "busybox", "true")
 
-	out, err := inspectField("test", "HostConfig.KernelMemory")
+	out, err := inspectField("test1", "HostConfig.KernelMemory")
 	c.Assert(err, check.IsNil)
 	if out != "52428800" {
```

Comment:
Note that this should really be `c.Assert(out, check.Equals, "52428800")`, it makes the error output nicer and doesn't need the extra comment.